interfaces/Portalicious/src/app/components/query-table/query-table.component.html

```diff
@@ -30,6 +30,7 @@
   [selectionMode]="enableSelection() ? 'multiple' : null"
   (selectionChange)="onSelectionChange($event)"
   (selectAllChange)="onSelectAllChange($event)"
+  (onFilter)="onFilterChange()"
```

Comment:
There is already a `onFilter` on line 12 :) 

I think you can just remove that one and keep this one instead.

Reply:
```suggestion
  (onFilter)="resetSelection()"
```

Actually I would avoid the extra function in the ts file tbh (especially given that the ts file is already quite large as it is)